client/my-sites/post-share/index.jsx

```diff
@@ -339,9 +351,24 @@ class PostShare extends Component {
 
 		return (
 			<div className="post-share__services">
-				<h5 className="post-share__services-header">
-					{ translate( 'Connected services' ) }
-				</h5>
+				<SectionHeader className="post-share__services-header" label={ translate( 'Connected accounts' ) }>
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'No social accounts connected' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/publicize-section-headings)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).